Libraries/LibJS/Print.cpp

```diff
@@ -1038,18 +1038,18 @@ ErrorOr<void> print_value(JS::PrintContext& print_context, JS::Value value, Hash
     else if (value.is_undefined())
         TRY(js_out(print_context, "\033[34;1m"));
 
-    if (value.is_string() && !print_context.disable_string_quotes)
+    if (value.is_string() && !print_context.raw_strings)
```

Comment:
I get what this commit does, but I don't understand why - maybe put the reason in the commit description?